torch/nn/quantized/functional.py

```diff
@@ -21,7 +17,7 @@ def conv2d(input, weight, bias,
            padding_mode='zeros',
            scale=1.0, zero_point=0,
            dtype=torch.quint8,
-           prepacked=True):
+           prepacked=False):
```

Comment:
Lets remove prepacked as an option, functionals by default should always do prepacking.